x/upgrade/keeper/keeper.go

```diff
@@ -189,6 +214,11 @@ func (k Keeper) ApplyUpgrade(ctx sdk.Context, plan types.Plan) {
 
 	handler(ctx, plan)
 
+	// incremement the protocol version and set it in state and baseapp
+	nextProtoVersion := k.GetProtocolVersion(ctx) + 1
+	k.setProtocolVersion(ctx, nextProtoVersion)
+	k.protoManager.SetProtocolVersion(nextProtoVersion)
```

Comment:
```suggestion
      if k.protoManager != nil {
	  k.protoManager.SetProtocolVersion(nextProtoVersion)
	}
```

we should have a safe default if this is `nil`